src/main/java/teammates/common/datatransfer/FeedbackContributionQuestionDetails.java

```diff
@@ -395,7 +395,8 @@ public String getQuestionResultStatisticsCsv(
         String csv = "";
         
         //Header
-        csv += "In the points given below, an equal share is equal to 100 points. ";
+        csv += Const.FEEDBACK_CONTRIBUTION_HEADER_BEFORE_COMMA;
```

Comment:
My intention was to combine "In the points given below" and "an equal share is equal to (...)"  that come after that in one single column because the comma is actually caught in a comma-separation algorithm.
Previously I didn't use constant, but if I don't if then the algorithm I wrote (csvToBeautifiedHtmlTable) will fail here if there are changes to the words "In the points given below", although by 'fail' I simply mean it won't be combined and beautified and really won't break anything, server-side or client-side.